pyproject.toml

```diff
@@ -20,9 +20,10 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-    "pyvista >= 0.43.0,<1",
+    "pyvista >= 0.45.0,<1",
```

Comment:
Wow... wait. This requirement might be too tough for some people. Why not leave the minimum version at 0.43?